jenkins_pipelines/uyuni_prs/tests_files/rubocop.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/bash
```

Comment:
@juliogonzalez  if we go opensource and the repo is public, we might run this via travis, i think we could consider it.
in

Reply:
IMHO we should use the same stuff for both Uyuni and SUSE manager, to avoid confusing people.

Reply:
ok